public_dropin_gpu_environments/nim_llm/dr_requirements.txt

```diff
@@ -12,54 +12,83 @@ anyio==4.6.2.post1
 argcomplete==1.11.1
 async-timeout==4.0.3
 attrs==24.2.0
+azure-core==1.32.0
+azure-identity==1.19.0
+azure-storage-blob==12.19.0
+boto3==1.35.53
+botocore==1.35.53
+cachetools==4.2.4
 certifi==2024.8.30
+cffi==1.17.1
 charset-normalizer==3.4.0
 click==8.1.7
+cryptography==43.0.3
 datarobot==3.5.2
-datarobot-drum==1.13.0
+datarobot-drum==1.14.1
 datarobot-mlops==10.2.0
 datarobot-mlops-connected-client==10.2.0
+datarobot-storage==0.0.0
```

Comment:
`datarobot-storage` pulls in all these new deps 🙃

Reply:
darn.